docs/reference/UDS Core/prerequisites.md

```diff
@@ -136,3 +140,180 @@ Metrics server is provided as an optional component in UDS Core and can be enabl
   optionalComponents:
     - metrics-server
 ```
+
+#### Loki
+
+The Loki deployment is (by default) backed by an object storage provider for log retention.  For cloud environments you can wire this into the environment's storage provider with the following overrides:
+
+```yaml
+loki:
+  values:
```

Comment:
nit: this confused me initially (especially when comparing against the minio example) - can we add the boilerplate of `overrides`, package, component, etc?

Reply:
I echo what @mjnagel said. It would be great to align these two examples.